lua/entities/gmod_wire_cpu.lua

```diff
@@ -4,8 +4,20 @@ ENT.PrintName       = "Wire ZCPU"
 ENT.Author          = "Black Phoenix"
 ENT.WireDebugName	= "ZCPU"
 
+CreateConVar("wire_cpu_max_frequency", "1400000", {FCVAR_REPLICATED})
+
 if CLIENT then return end -- No more client
 
+local cpu_max_frequency = nil
```

Comment:
Instead of calling `updateCPUMaxFrequency()` on the line 18, set this to the default value and move the ConVar creation below this line.